packages/hdwallet-tally/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "isolatedModules": false
```

Comment:
Why turn this off?
```suggestion
    "outDir": "dist"
```

Reply:
It was to fix linting issues. I'm still learning. I had forgotten to change it back after finding the real fix.

Reply:
Sorry if this came across as brusque! I appreciate your efforts, and you're doing fine.